src/App.vue

```diff
@@ -722,8 +723,24 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
         throw error // go to catch()
       }
 
-      // FUTURE: since we know what type of filing this is,
-      // add staff check for certain filings (ie, restorations)
+      // get Keycloak roles
+      const keycloakRoles = await this.loadKeycloakRoles().catch(error => {
+        console.log('Keycloak roles error =', error) // eslint-disable-line no-console
+        this.accountAuthorizationDialog = true
+        throw error // go to catch()
+      })
```

Comment:
Need to get roles before we can check if this is a staff user.

I moved it here, above getting the user info, because we don't need to fetch the user info if the staff check fails.

Currently, the staff check is disabled. I am not sure if Continuation In is for staff only or regular users. I will ask.

Reply:
This is still an outstanding question so I'll leave the code commented out for now.